### ISSUE TITLE
Bad alignment

### ISSUE THREAD

**Issue by cbarrick** [esc/conda-zsh-completion#5]
I am seeing bad alignment in my conda completions when using menu selection. I really have no idea how this works though, so I'm not sure if this is the completion's fault or zsh.

I am using zsh v5.3.1 on macOS 10.13 beta:

```
% zsh --version
zsh 5.3.1 (x86_64-apple-darwin17.0.0)
```

The relevant section of my zshrc:

```
# Completion
#--------------------
autoload -Uz compinit && compinit -u

zstyle ':completion:*' use-cache true # Cache completion to `${ZDOTDIR}/.zcompcache`
zstyle ':completion:*' squeeze-slashes true # Strip slashes from directories
zstyle ':completion:*' menu select # Make the menu interactive with arrow keys

bindkey '^I' menu-expand-or-complete
bindkey '^[[Z' reverse-menu-complete
```

The results I am seeing:

```
% conda build
build  -- tool for building conda packages
bundle   -- Create or extract a "bundle package" (EXPERIMENTAL)
clean   -- Remove unused packages and caches.
config  -- Modify configuration values in .condarc.
create     -- Create a new conda environment from a list of specified packages.
env    -- Manage environments.
help  -- Displays a list of available conda commands and their help strings.
info       -- Display information about current conda install.
init     -- Initialize conda into a regular environment. (DEPRECATED)
install  -- Install a list of packages into a specified conda environment.
list       -- List linked packages in a conda environment.
package  -- Low-level conda package utility. (EXPERIMENTAL)
remove     -- Remove a list of packages from a specified conda environment.
run      -- Launches an application installed with Conda.
search   -- Search for packages and display their information.
uninstall  -- Alias for conda remove
update  -- Update conda packages.
```

<img width="1022" alt="_conda" src="https://user-images.githubusercontent.com/271982/29393967-45823e9a-82d5-11e7-8836-7499d1ab99d0.png">

**Comment by esc** [comment]
Somehow your zsh isn't picking up on the categories, here is what it looks like on my Linux box:


![2019-03-09-050944_764x481_scrot](https://user-images.githubusercontent.com/41563/54066046-a7557a00-4229-11e9-9fe5-0f7dce515763.png)


Using zsh version `zsh 5.4.2 (x86_64-ubuntu-linux-gnu)`

**Comment by esc** [comment]
There is a new potential fix in #13 here is the screenshot:

![2019-03-09-055032_815x489_scrot](https://user-images.githubusercontent.com/41563/54066393-5183d080-422f-11e9-9901-45189fa90181.png)

@cbarrick can you check if this solves the issue for you too?

**Comment by esc** [comment]
New attempt in #13

**Comment by esc** [comment]
Solved by #15
